lib/goodreads/goodreads.js

```diff
@@ -2,17 +2,18 @@
 /* Goodreads - Handles all connectivity to Goodreads API */
 /* API Docs: http://www.goodreads.com/api */
 
-const http = require('http');
-const xml2js = require('xml2js');
-const oauth = require('oauth').OAuth;
-const utils = require('utils');
+import http from 'http';
+
+import xml2js from 'xml2js';
```

Comment:
please keep these as require statements, that way it can be run without precompile on Node 6